### ISSUE TITLE
2025-12-30: vllm-project/vllm 仓库更新报告

### ISSUE THREAD

**Issue by yan97ao** [yan97ao/argus#559]
# 每日更新报告（2025-12-30）

## vllm-project/vllm

| 提交时间 | 作者 | 提交信息 |
|----------|------|----------|
| 2025-12-30 20:34:08 | Li, Jiang | [CPU] Disable async schedule on CPU (#31525) |
| 2025-12-30 20:26:12 | Nicolò Lucchesi | [CI][NIXL] Split DPEP tests (#31491) |
| 2025-12-30 17:21:49 | Pleaplusone | [ROCm][Bugfix] Fix accuracy issue on fmoe when `VLLM_ROCM_USE_AITER_FUSION_SHARED_EXPERTS` enabled (#31523) |
| 2025-12-30 15:21:13 | Kevin | [Frontend] add continue_final_message parameter to /embeddings endpoint (#31497) |
| 2025-12-30 14:52:28 | Roger Feng | [xpu] [bugfix] upgrade to latest oneccl in dockerfile (#31522) |
| 2025-12-30 14:42:06 | Nick Hill | [Minor] Various small code cleanups/simplifications (#31508) |
| 2025-12-30 14:27:31 | ZT-AIA | Add Loraconfig parameter to  get_punica_wrapper function (#31408) |
| 2025-12-30 12:26:15 | Roy Wang | Migrate meetups & sponsors [2/N] (#31500) |
| 2025-12-30 11:38:47 | Hojin Yang | feat(frontend): add --default-chat-template-kwargs CLI argument (#31343) |
| 2025-12-30 10:42:45 | Nick Hill | [Core] Deduplicate generate/encode logic in `AsyncLLM` (#31510) |
| 2025-12-30 10:01:58 | wangln19 | fix: update kimi k2 tool parser logic (#31207) |
| 2025-12-30 08:17:16 | Sage | [Prefix Cache] Include lora_name in BlockStored event for deterministic KV-cache reconstruction (#27577) |
| 2025-12-30 07:48:56 | qli88 | [CI]Test Group 'NixlConnector PD accuracy tests' is fixed (#31460) |
| 2025-12-30 05:53:59 | Alexei-V-Ivanov-AMD | [CI/ROCm] Fixing "V1 Test attention (H100)" test group. (#31187) |
| 2025-12-30 05:27:55 | Robert Shaw | [Bugfix][ROCm] Fix Static Quant Issue (#31502) |
| 2025-12-30 05:27:00 | Robert Shaw | [MoE Refactor][12/N] Marlin Fp8 MoE Pure Function (#31499) |
| 2025-12-30 04:20:55 | Nick Hill | [Core] Enable async scheduling by default (#27614) |
| 2025-12-30 03:13:42 | chunxiaozheng | implements register kv caches in lmcache connector (#31397) |
| 2025-12-30 00:30:18 | Roger Young | Optimize QKNorm for MiniMax-M2/M2.1 (#31493) |
| 2025-12-30 00:22:39 | RickyChen / 陳昭儒 | [Feature] Add offline FastAPI documentation support for air-gapped environments (#30184) |
| 2025-12-30 00:20:01 | Harry Mellor | Replace `nn.ConvNd` with vLLM's `ConvNdLayer` for Transformers modeling backend (#31498) |
